wasabi/tables.py

```diff
@@ -126,16 +127,17 @@ def row(
         and not env_log_friendly
         and (fg_colors is not None or bg_colors is not None)
     )
-    cols = []
-    if isinstance(aligns, basestring_):  # single align value
-        aligns = [aligns for _ in data]
+    cols: List[str] = []
+    _aligns = (
+        [aligns for _ in data] if isinstance(aligns, str) else cast(List[str], aligns)
```

Comment:
There are some value re-assignment here, so I opted to just type-cast the `aligns` method and reassign its value to another variable.